Neos.Neos/Documentation/CreatingASite/NodeTypes/DynamicConfigurationProcessing.rst

```diff
@@ -0,0 +1,77 @@
+.. _dynamic-configuration-processing:
+
+============================================
+Dynamic Client-side Configuration Processing
+============================================
+
+.. note:: This API is rather low-level and still experimental, we might change
+   some of the implementation details or compliment it with a more high-level API.
+
+All configuration values that begin with ``ClientEval:`` are dynamically evaluated on
+the client side. They are written in plain JavaScript (evaluated with ``eval``) and
+have ``node`` variable in the scope pointing to the currently focused node, with all
+transient inspector changes applied. For now it is only related to the nodetypes
+inspector configuration, but in the future may be extended to the other parts of
+the user interface.
+
+
+A few Practical Examples
+========================
+
+Hiding one property when other is not set
```

Comment:
> Hiding one property when another one is not set 

would IMO be better

Reply:
Changed to "Hiding one property when the other one is not set"